docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -803,20 +800,18 @@ The acceptable flags include all those of `kubectl` except for
 - `-X`: turn on debug echoing of the commands inside the script that
   implements this command.
 
+The space manager is involved in the creation of the workload management
+space. To access the space manager, the following environment variables
+are used: SM_CONFIG specifies the path to the space manager kubeconfig 
+file. SM_CONTEXT specifies the context of the space manager within the 
+kubeconfig file. IN_CLUSTER specifies whether the space manager is
+accessed from within the hosting cluster or externally. Note that 
```

Comment:
Merely speaking of "is accessed" without specifying _who_ is doing the access or _when_ makes this a universal statement. But I think it is not intended to be, it is intended to be a statement just about the access involved in this command. So I would reword this to say something like the following.

> IN_CLUSTER is `true` or `false` depending on whether the command is being issued from within the KubeStellar host cluster.

But that is not exactly right either, because there does not have to be a KubeStellar host cluster. So a full explanation would be more like the following.
> IN_CLUSTER indicates the networking relationship between the invoker of this command and the Space Manager. When there is a KubeStellar host cluster, `IN_CLUSTER=true` means that the invoker and manager are in the host cluster, `false` means the invoker is outside the host cluster. When the central components are run as bare processes, `true` means the "native" or "internal" kubeconfig of the relevant spaces will be used, `false` means the "external" kubeconfig of the relevant spaces will be used.